pipeline/tests/database_test.py

```diff
@@ -21,6 +21,7 @@
 def test_table_counts():
     test.assert_table_count("community_area", 77)
     test.assert_table_count("population", 770)
+    test.assert_table_count("income", 770)
```

Comment:
Thanks for adding these, will help make sure future changes to the database have the right tables.